docs/changelog/README.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- [serverside-collection] checkboxes in table are toggleable with an `if`-statement
```

Comment:
=> properly pass `show-select` attribute to `v-data-table` in order to make checkboxes toggleable.